modules/loader-utils/src/lib/files/blob-file.ts

```diff
@@ -30,8 +30,10 @@ export class BlobFile implements ReadableFile {
     };
   }
 
-  async read(start: number, length: number): Promise<ArrayBuffer> {
-    const arrayBuffer = await this.handle.slice(start, start + length).arrayBuffer();
+  async read(start?: number | BigInt, length?: number): Promise<ArrayBuffer> {
+    const arrayBuffer = await this.handle
```

Comment:
Why this change?

Reply:
I faced a problem when I have `ReadableFile` as input in `FileHandler` and read() there accepts `start` as `number | bigint` so I pass bigint here and got runtime error that it cannot sum bigint and number because real implementation knows nothing about bigint. We need keep real implementation of the methods matching it's description in `ReadableFile`